tsuru/client/services.go

```diff
@@ -469,6 +469,32 @@ func (c ServiceInstanceInfo) Run(ctx *cmd.Context, client *cmd.Client) error {
 			}
 		}
 	}
+
+	url, err = cmd.GetURL("/services/" + serviceName + "/instances/" + instanceName + "/status")
+	if err != nil {
+		return err
+	}
+	request, err = http.NewRequest("GET", url, nil)
+	if err != nil {
+		return err
+	}
+	resp, err = client.Do(request)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	bMsg, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+	msg := string(bMsg) + "\n"
```

Comment:
Could you add a "Status:" label here, something like `fmt.Sprintf("Status: %s\n", bMsg)`?

Reply:
Updated the output here.